CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 ## Bug Fixes
 
+* **css:** Notification bar padding of links only working for complete sentences (#1000)
```

Comment:
ah, whoops, needs a rebase to fix the merge conflict (sorry for the slow review)

Reply:
Sure thing, updated 🚀 ~ Yea I usually don't keep on top of the changelog conflicts until the time comes for the PR, so just hint at what should go in next, and I'll make sure it can merge at that point. :shipit: 🚢